test/sasdataloader/test/utest_abs_reader.py

```diff
@@ -357,7 +357,7 @@ def test_slits(self):
         self.assertEqual(self.data.dxw[0], 0.001)
         self.assertEqual(self.data.dy[0], 3)
         self.assertEqual(self.data.x[1], 0.03)
-        self.assertAlmostEquals(self.data.y[1], 1001.0)
+        self.assertAlmostEqual(self.data.y[1], 1001.0)
```

Comment:
Three files have merge conflicts; cansas_reader_HDF5.py, utest_abs_reader.py, and utest_generic_file_reader_class.py. This line (360 in utest_abs_reader) is the only line that should stay from this branch for those 3 files. The remaining changes for the three files have already been incorporated into master, along with other changes that supersede what was done here.